utils/data_helpers.py

```diff
@@ -56,6 +57,18 @@ def create_tfrecords(dataset_type, tfrecords_dir, image_source_dir, text_source_
     else:
         raise Exception(f'{dataset_type} is not a recognised dataset type')
 
+def extract_image_bounding_boxes(image_filenames, base_path='data/CUB_200_2011_with_text/images/CUB_200_2011'):
+    '''
+    Returns a map of filename to bounding box in format [x-top, y-top, w, h]
+    '''
```

Comment:
Use double quotes on docstrings to be consistent

Reply:
Haha my bad, I'll change it in next PR